packages/core-contracts/contracts/token/ERC721Owned.sol

```diff
@@ -0,0 +1,37 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./ERC721.sol";
+import "../ownership/Ownable.sol";
+
+contract ERC721Owned is ERC721, Ownable {
+    function transferFrom(
+        address from,
+        address to,
+        uint256 tokenId
+    ) public virtual override onlyOwner {
+        super.transferFrom(from, to, tokenId);
+    }
+
+    function safeTransferFrom(
+        address from,
+        address to,
+        uint256 tokenId
+    ) public virtual override onlyOwner {
+        super.safeTransferFrom(from, to, tokenId);
+    }
+
+    function safeTransferFrom(
+        address from,
+        address to,
+        uint256 tokenId,
+        bytes memory data
+    ) public virtual override onlyOwner {
+        super.safeTransferFrom(from, to, tokenId, data);
+    }
+
+    function _isApprovedOrOwner(address, uint256) internal view virtual override onlyOwner returns (bool) {
```

Comment:
Hmmm, not sure about this one. It's weird to have a view function with an access modifier. I'd prefer if it had no modifier and returned `address == owner`.

Reply:
agreed, it can be used inside of a modifier instead fi we want this check

Reply:
Comment addressed. Please confirm this is better (@ajsantander ). Thanks